src/olympia/devhub/views.py

```diff
@@ -608,13 +608,10 @@ def check_addon_compatibility(request):
                        None, None, request=request)})
 
 
-def handle_upload(filedata, user, app_id=None, version_id=None, addon=None,
-                  is_standalone=False, is_listed=True, automated=False,
-                  submit=False):
-    if addon:
-        # TODO: Handle betas.
-        automated = addon.automated_signing
-        is_listed = addon.is_listed
```

Comment:
Wow, that was confusing. I was wondering why `signing/views.py` was calling `handle_upload()` without changing `is_listed`... Glad you're changing this.